src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6274,8 +6274,8 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_InvalidWithReceiverType" xml:space="preserve">
     <value>The receiver of a `with` expression must have a non-void type.</value>
   </data>
-  <data name="ERR_NoSingleCloneMethod" xml:space="preserve">
-    <value>The receiver type '{0}' is not a valid record type.</value>
+  <data name="ERR_CannotClone" xml:space="preserve">
+    <value>The receiver type '{0}' is not a valid record type and is not a struct type.</value>
```

Comment:
Consider adding a `<comment>` that 'record' and 'struct' are language keywords for loc team

Reply:
In general, I'm not sure about such comments. In this instance, they feel wrong. `record` and `struct` are not keywords in this sentence.
If they were keywords, then they could be marked with backticks in markdown, but that doesn't work here: "...is not a valid `record` type and is not a `struct` type."

---
In reply to: [608248749](https://github.com/dotnet/roslyn/pull/52319#discussion_r608248749) [](ancestors = 608248749)